Dockerfile

```diff
@@ -5,11 +5,11 @@ WORKDIR /crawler
 COPY requirements.txt /crawler/requirements.txt
 RUN pip install -r requirements.txt
 
-ADD crawler /crawler
-
 COPY dependencies/python-socket-datacollector_0.1.1-1_all.deb /tmp
 RUN dpkg -i /tmp/python-socket-datacollector_*_all.deb && \
     apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get -y install fprobe
+    DEBIAN_FRONTEND=noninteractive apt-get -y install softflowd
+
+ADD crawler /crawler
```

Comment:
Is this what you mean by 'making container build more efficient by reversing order of cmds' ?
Why is this more efficient?